apps/meteor/ee/app/livechat-enterprise/server/api/inquiries.ts

```diff
@@ -1,19 +1,24 @@
 import { API } from '../../../../../app/api/server';
-import { setPriorityToInquiry } from './lib/inquiries';
+import { setSLAToInquiry } from './lib/inquiries';
 
 API.v1.addRoute(
 	'livechat/inquiry.prioritize',
```

Comment:
Since this endpoint wont "prioritize" anymore, the name may be a bit misleading. 

What about `inquiriy.setSLA`? 👀

Reply:
changed